linker/Mono.Linker.Steps/MarkStep.cs

```diff
@@ -461,12 +483,12 @@ protected void MarkAssembly (AssemblyDefinition assembly)
 
 			MarkAssemblySpecialCustomAttributes (assembly);
 
-			MarkCustomAttributes (assembly);
+			LazyMarkCustomAttributes (assembly);
```

Comment:
@marek-safar Is this what you had in mind?  https://github.com/Unity-Technologies/linker/commit/d2837cdead9f41461f80f50ab455ab87b743cf71